packages/react-devtools-shared/src/constants.js

```diff
@@ -316,9 +318,11 @@ export const THEME_STYLES: {[style: Theme | DisplayDensity]: any} = {
     '--color-scheduling-profiler-react-suspense-resolved-hover': '#89d281',
     '--color-scheduling-profiler-react-suspense-unresolved': '#c9cacd',
     '--color-scheduling-profiler-react-suspense-unresolved-hover': '#93959a',
+    '--color-scheduling-profiler-thrown-error': '#fb3655',
+    '--color-scheduling-profiler-thrown-error-hover': '#f82042',
     '--color-scheduling-profiler-text-color': '#282c34',
     '--color-scheduling-profiler-text-dim-color': '#555b66',
-    '--color-scheduling-profiler-react-work-border': '#ffffff',
+    '--color-scheduling-profiler-react-work-border': '#3d424a',
```

Comment:
These color values were broken before. Noticed while implementing this so I fixed them too.